tt_metal/core_descriptors/blackhole_140_arch.yaml

```diff
@@ -6,30 +6,31 @@
 #     core descriptor config
 
 blackhole:
-  1:
-    compute_with_storage_grid_range: # Logical only start and end [x, y]
-      start: [0, 0]
-      end: [12, 9]
+  row:
```

Comment:
for BH we do column dispatch only

Reply:
if we change default to col, then some files need to select based on arch for default, and for some files arch seems not accessible. 
what if we change the enum DispatchCoreAxis to default, col?

Reply:
for example, in rtoptions.hpp I don't see where to access arch

Reply:
the arch can always be accessed by `get_platform_architecture`